hedera-node/test-clients/src/main/java/com/hedera/services/bdd/suites/crypto/CryptoApproveAllowanceSuite.java

```diff
@@ -175,7 +174,6 @@ private HapiSpec cannotPayForAnyTransactionWithContractAccount() {
                         .hasPrecheck(PAYER_ACCOUNT_NOT_FOUND)));
     }
 
-    @HapiTest
```

Comment:
We should not be removing these.
If the test was passing and now fails, then we need to find out why and get that fixed (either here or with a filed issue).

Reply:
Okay, let's debug in another PR, to keep this one focused.